pkgs/development/tools/mpfshell/default.nix

```
@@ -1,14 +1,14 @@
 { lib, python3Packages, fetchFromGitHub }:
 
 python3Packages.buildPythonPackage rec {
-  name = "mpfshell-${version}";
-  version = "0.8.1";
+  pname = "mpfshell-unstable";
+  version = "2020-04-11";
```

```suggestion
  pname = "mpfshell";
  version = "unstable-2020-04-11";
```
Pls set `unstable` as a part of version

---

Version must start with a digit so unstable needs to be part of `pname`: https://nixos.org/manual/nixpkgs/stable/#sec-package-naming

---

I'm not sure that is still up to date since we've moved to `pname`.

---

https://github.com/NixOS/nixpkgs/issues/68518

---

Please apply this review comment.

---

done